blog/2024-08-19-building-chaindensity.md

```diff
@@ -6,7 +6,7 @@ slug: /building-chaindensity
 
 <img src="/blog-assets/density1.png" alt="Cover Image for Building ChainDensity" width="100%"/>
 
-# Building ChainDensity: Unveiling Blockchain Activity Patterns with HyperSync
+<!--truncate-->
```

Comment:
Do we no longer want the heading? Happy if you happy.

Reply:
I didn't mean to erase the heading, there's still a heading as the  `title` config on line 2 becomes the heading and what that field is for. Then there's also 'Introduction: The Genesis of Chain Density" , see reference pic below on preview.

<img width="857" alt="Screenshot 2024-09-06 at 11 27 02" src="https://github.com/user-attachments/assets/c43266a6-314c-443d-b42c-6adfa5271299">

I'll change the title config on line 2 to: "Building ChainDensity: Unveiling Blockchain Activity Patterns with HyperSync"